src/main/java/design/MazeDesignerInteractor.java

```diff
@@ -0,0 +1,67 @@
+package design;
+
+import entities.DesignableMaze;
+import javafx.scene.control.Button;
+import javafx.stage.Stage;
+
+public class MazeDesignerInteractor implements MazeDesignerInputBoundary {
+
+    DesignableMaze dm;
+    RandomizedPrim mg;
+    private MazeDesignerOutputBoundary mdp;
+
+
+    public MazeDesignerInteractor(MazeDesignerOutputBoundary mdp) {
+        // update designablemaze instance attribute
+        newMaze();
+        resetMaze();
+        // update the randomizedprim instance attribute
+        this.mg = new RandomizedPrim();
+        // update the output boundary instance attribute
+        this.mdp = mdp;
+    }
+
+    public void newMaze(){
+        dm = new DesignableMaze(11, 17);
```

Comment:
since you update dm with a new DesignableMaze instance, you also need to call 

`mg = new RandomizedPrim(dm)`